lib/linguist/languages.yml

```diff
@@ -5067,6 +5068,9 @@ Shell:
   - ".bash_profile"
   - ".bashrc"
   - ".cshrc"
+  - ".env"
+  - ".env.example"
+  - ".flaskenv"
```

Comment:
Could you please update the template with a search query for this filename?

Reply:
Sure. Done.

Reply:
Thanks!